gtda/base.py

```diff
@@ -145,5 +145,6 @@ def transform_plot(self, X, sample=0, **plot_params):
 
         """
         Xt = self.transform(X[[sample]])
-        self.plot(Xt, sample=sample, **plot_params)
+        self.plot(Xt, sample=0, **plot_params)
```

Comment:
`sample=0` seems incorrect to me.

Reply:
My interpretation is as follows:
we transform **only** the sample we want, so, `Xt.shape=(1, ...)`.
I mean, we have already selected that sample,  and now, we want to plot the first that is remaining.
what do you think?

Reply:
You are quite right! I was wrong before.